### PR TITLE
roachtest: ignore 3 flaky ruby tests

### LINKED ISSUE #127612: roachtest: ruby-pg failed
roachtest.ruby-pg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/16180468?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/16180468?buildTab=artifacts#/ruby-pg) on release-24.2 @ [60d3fb84a73f5e31c99404223f3a0ba91e67f489](https://github.com/cockroachdb/cockroach/commits/60d3fb84a73f5e31c99404223f3a0ba91e67f489):

```
(orm_helpers.go:229).summarizeFailed: 
Tests run on Cockroach v24.2.0-beta.2-dev-60d3fb84a73f5e31c99404223f3a0ba91e67f489
Tests run against ruby-pg v1.4.6
898 Total Tests Run
713 tests passed
185 tests failed
0 tests skipped
2 tests ignored
2 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
2 tests expected failed but not run
---
--- FAIL: PG::Connection times out after connect_timeout seconds -  (unexpected)
For a full summary look at the ruby-pg artifacts 
An updated blocklist (rubyPGBlocklist) is available in the artifacts' ruby-pg log
test artifacts and logs in: /artifacts/ruby-pg/cpu_arch=arm64/run_1
```

Parameters:
 - <code>ROACHTEST_arch=arm64</code>
 - <code>ROACHTEST_cloud=gce</code>
 - <code>ROACHTEST_coverageBuild=false</code>
 - <code>ROACHTEST_cpu=4</code>
 - <code>ROACHTEST_encrypted=false</code>
 - <code>ROACHTEST_metamorphicBuild=false</code>
 - <code>ROACHTEST_ssd=0</code>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-16180468/ruby-pg/1721659729466/1721660433234)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127329 roachtest: ruby-pg failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.1]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*ruby-pg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

Jira issue: CRDB-40514

### PR DESCRIPTION
**Pull request by annrpom**
Commits are separated out in anticipation of a backport to 24.1

Fixes: #127612
Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/tests/ruby_pg_blocklist.go [hunk 1]
```diff
@@ -204,6 +204,9 @@ var rubyPGIgnorelist = blocklist{
 	`PG::Connection handles server close while asynchronous connect`:                                                                                                               "flaky",
 	`PG::Connection OS thread support Connection.new shouldn't block a second thread`:                                                                                              "flaky",
 	`PG::Connection multinationalization support respect and convert character encoding of input strings should convert error string to #put_copy_end`:                             "unknown",
+	`PG::Connection in nonblocking mode rejects to send lots of COPY data`:                                                                                                         "flaky",
```
Comment (rafiss) [inline pkg/cmd/roachtest/tests/ruby_pg_blocklist.go:207]: iet's also make sure to remove these from the `blocklist` if they are present

**Review by rafiss — APPROVED** [review]

File: pkg/cmd/roachtest/tests/ruby_pg_blocklist.go (hunk 1 above)
Comment (annrpom) [reply pkg/cmd/roachtest/tests/ruby_pg_blocklist.go:205]: ah thank u for the catch

**Comment by annrpom** [comment]
TFTR! ('-')7 

bors r=rafiss

**Comment by rafiss** [comment]
blathers backport 24.2

**Comment by rafiss** [comment]
blathers backport 23.2 23.1

**Comment by rafiss** [comment]
blathers backport 23.1

**Pull request merged**